From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Thu, 25 Oct 2018 21:55:23 +0200
Subject: [PATCH] {BUG in 4.20-rc1} drivers/scsi/ufs/ufs_bsg: silence
 -Wunused-but-set-parameter
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

gcc 8.2.1 reports:

    drivers/scsi/ufs/ufs_bsg.c: In function ‘ufs_bsg_verify_query_params’:
    drivers/scsi/ufs/ufs_bsg.c:51:21: error: parameter ‘desc_buff’ set
    but not used [-Werror=unused-but-set-parameter]
            uint8_t *desc_buff, int *desc_len,
            ~~~~~~~~~^~~~~~~~~
---
 drivers/scsi/ufs/ufs_bsg.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/scsi/ufs/ufs_bsg.c b/drivers/scsi/ufs/ufs_bsg.c
index 775bb4e5e36e..dafe3c447dc1 100644
--- a/drivers/scsi/ufs/ufs_bsg.c
+++ b/drivers/scsi/ufs/ufs_bsg.c
@@ -71,7 +71,7 @@ static int ufs_bsg_verify_query_params(struct ufs_hba *hba,
 				      desc_op))
 		return -EINVAL;
 
-	desc_buff = (uint8_t *)(bsg_request + 1);
+	/* desc_buff = (uint8_t *)(bsg_request + 1); BUG? Unused variable */
 
 out:
 	return 0;
-- 
